notes.md

```diff
@@ -95,25 +95,37 @@ Overlay tracks should be rendered in the same 'channel' as the track it's linked
 
-### Segment linking
+### Linked Segments
 
-#### Hard linking
+Matroska provides several methods to link two or many Segments together to create a Linked Segment. A Linked Segment is a set of multiple Segments related together into a single presentation by using Hard Linking, Soft Linking, or Medium Linking. All Segments within a Linked Segment MUST utilize the same track numbers and timescale. All Segments within a Linked Segment MUST be stored within the same directory. All Segments within a Linked Segment MUST store a `SegmentUID`.
 
-This linking can also be called splitting. It's the operation of cutting one Segment in several parts. The resulting parts should play as if it was just one part (the original Segment). That means the timecode of each part follows the ones from the previous parts. The track numbers are the same. The chapters only match the current Segment (unless the edition is ordered, where all parts should be in each Segment). And most important, the NextUID and PrevUID points the respective Segment UIDs.
+#### Hard Linking
 
-#### Soft linking
+Hard Linking (also called splitting) is the process of creating a Linked Segment by relating multiple Segments using the PrevUID and NextUID Elements. Within a Linked Segment the timecode of each Segment MUST follow consecutively in linking order. With Hard Linking, the chapters of any Segment within the Linked Segment MUST only reference the current Segment. With Hard Linking, the NextUID and PrevUID MUST reference the respective Segment UIDs of the next and previous Segments. The first Segment of a Linked Segment MUST have a NextUID Element but not a PrevUID Element. The last Segment of a Linked Segment MUST have a PrevUID Element but not a NextUID Element. The middle Segments of a Linked Segment MUST have both a NextUID Element and a PrevUID Element. 
 
-Soft linking is used by codec chapters. They can reference another Segment and jump on that Segment. The way the Segments are described are internal to the chapter codec and unknown to the Matroska level. But there are Elements in the Segment Information (such as `ChapterTranslate`) that can translate a value representing a Segment in the chapter codec and to the current Segment UID. All Segments that could be used in a file/segment this way should be marked as members of the same family (SegmentFamily), so that the player can quickly switch from one to the other.
+As an example four Segments may be Hard Linked as a Linked Segment through cross-referencing each other with `SegmentUID`, `PrevUID`, and `NextUID` as in this table.
 
-#### Medium linking
+file name  | SegmentUID                       | PrevUID                          | NextUID
```

Comment:
The element names should use `backticks`. The file names could use them as well.